crates/router/src/events.rs

```diff
@@ -81,3 +82,21 @@ impl EventsHandler {
         };
     }
 }
+
+impl MessagingInterface for EventsHandler {
+    type MessageClass = EventType;
+
+    fn send_message<T>(
+        &self,
+        data: T,
+        timestamp: PrimitiveDateTime,
+    ) -> error_stack::Result<(), EventsError>
+    where
+        T: Message<Class = Self::MessageClass> + Serialize,
```

Comment:
This should be ErasedMaskedSerialize instead of Serialize.

Reply:
updated